charts/cinder-csi-plugin/values.yaml

```diff
@@ -75,3 +75,12 @@ storageClass:
   retain:
     isDefault: false
     allowVolumeExpansion: true
+
+priorityClassName:
+  controllerPlugin: ""
+  nodePlugin: ""
+
+podSecurityPolicy:
+  enabled: false
```

Comment:
just curious, is this for Helm only or both for manifest ? or should we update 
https://github.com/kubernetes/cloud-provider-openstack/tree/master/manifests/cinder-csi-plugin
as well?

Reply:
I don't think it's appropriate to add this in the yaml manifest as this is totally optional.

Reply:
yeah, basically Helm should be more flexible, and manifest is just for testing purpose IMO.